src/item/Item.js

```diff
@@ -4347,6 +4347,16 @@ new function() { // Injection scope for hit-test functions shared with project
             // Apply the parent's global matrix to the calculation of correct
             // bounds.
             var bounds = this.getStrokeBounds(viewMatrix);
+            // Account for drop shadow
+            var style = this.getStyle();
+            if (style.hasShadow()) {
+                var shadowRadius = style.getShadowBlur() * Numerical.GAUSSIAN_BLUR_COEFFICIENT;
```

Comment:
We don't really need to put this into `Numerical.GAUSSIAN_BLUR_COEFFICIENT` since it's only used in one place.

Could you inline the value directly here, and add the comment about the reason right above it?

Reply:
@georeith still waiting on feedback from you here

Reply:
@lehni sorry I forgot to come back to this just seen it pop up in my feed again. I can make this change.

Reply:
That'd be great, thank you!